### PR TITLE
docs(geo): Forced Metadata documentation and examples

### PR DESCRIPTION
**Pull request by mneige**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Documentation doesn't cover the new Forced Metadata feature

**What is the new behavior?**
Documentation and examples are now provided in ReadTheDocs

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications:

**Other information**:

### CODE REVIEW

File: docs/_tables/fr/config/catalog.csv [hunk 1]
```diff
@@ -1,37 +1,41 @@
...
+**id*** ,String ,Identifiant unique permettant de différencier les catalogues entre eux. ,,uuid()
 groupImpose ,"| id*: String,
 | title*: String ","| N.B: Propriété disponible sur un objet de type CompositeCatalog
 | Permet d'imposer l'utilisation d'un groupe à l'ensemble des couches appellées du | catalogue.
 |                - id: Identifiant unique permettant de différencier les groupes entre eux.
 |                - title: Titre pour le groupe qui sera utilisé dans l'outil Catalog.",,
 forcedProperties ,"| layerName*: String,
-| title*: String ","| Permet d'imposer un nom à une couche spécifique.
+| title*: String, metadataUrl: String, metadataAbstract: String, metadataAbstractAll: String, metadataUrlAll: String ","| Permet d'imposer un nom à une couche spécifique.
 |               - layerName: Nom de la couche dans le catalogue d'origine (propriété layer name). Permet d'aller chercher la couche en question dans le service.
```
Comment (pelord) [inline docs/_tables/fr/config/catalog.csv:12]: je ne suis pas sur que c'est dans le bon format.....

File: src/environments/environment.ts [hunk 2]
```diff
@@ -124,18 +124,29 @@ export const environment: Environment = {
...
+              forcedProperties: [
+                {
+                  newMetadataUrl: 'https://www.donneesquebec.ca/',
+                  layerName: 'lieuhabite',
+                  title: 'Nouveau nom pour cette couche WMS et nouvel url pour toutes les couches',
+                  //metadataAbstract: "New WMS Abstract",
+                  //metadataUrl: "https://gitlab.com/",
+                },
+              {
+                layerName: "All",
```
Comment (pelord) [inline src/environments/environment.ts:145]: Je ne penses pas que ceci reflète ton code dans la lib 
image

Comment (pelord) [reply src/environments/environment.ts:145]: Que fait-on si  un layer se nomme All?
 allons y plutot avec *  ?

File: src/environments/environment.ts [hunk 3]
```diff
@@ -124,18 +124,29 @@ export const environment: Environment = {
...
               id: 'forcedProperties_wms',
               url: 'https://ws.mapserver.transports.gouv.qc.ca/swtq',
               type: 'wms',
-              forcedProperties: [{
-                layerName: 'lieuhabite',
-                title: 'Nouveau nom pour cette couche WMS'
-              }]
+              forcedProperties: [
+                {
+                  newMetadataUrl: 'https://www.donneesquebec.ca/',
```
Comment (pelord) [inline src/environments/environment.ts:138]: newMetadataUrl ne semble pas fonctionner....

**Review by pelord — CHANGES_REQUESTED** [review]

File: src/app/pages/portal/portal.component.scss [hunk 4]
```diff
@@ -77,6 +77,10 @@ igo-map-browser.sidenav-offset-baselayers igo-baselayers-switcher {
   left: 405px;
 }
 
+igo-baselayers-switcher {
```
[the remark this answers is not in the document]
Comment (mneige) [reply src/app/pages/portal/portal.component.scss:80]: Je n'y ai pas touché

**Review by PhilippeLafreniere18 — APPROVED** [review]

**Review by PhilippeLafreniere18 — APPROVED** [review]

**Pull request merged**